examples/nirfmxnr/modacc-noncontiguous-multicarrier.py

```diff
@@ -0,0 +1,738 @@
+            )
+        )
+
+        client.SetAttributeI32(
+            nirfmxnr_types.SetAttributeI32Request(
+                instrument=instr,
+                selector_string=subblock_string,
+                attribute_id=nirfmxnr_types.NIRFMXNR_ATTRIBUTE_COMPONENT_CARRIER_SPACING_TYPE,
+                attr_val_raw=COMPONENT_CARRIER_SPACING_TYPE[i],
+            )
+        )
+
+        client.SetAttributeI32(
+            nirfmxnr_types.SetAttributeI32Request(
+                instrument=instr,
+                selector_string=subblock_string,
+                attribute_id=nirfmxnr_types.NIRFMXNR_ATTRIBUTE_COMPONENT_CARRIER_AT_CENTER_FREQUENCY,
+                attr_val_raw=COMPONENT_CARRIER_AT_CENTER_FREQUENCY[i],
+            )
+        )
+
+        client.SetAttributeF64(
+            nirfmxnr_types.SetAttributeF64Request(
+                instrument=instr,
+                selector_string=subblock_string,
+                attribute_id=nirfmxnr_types.NIRFMXNR_ATTRIBUTE_CHANNEL_RASTER,
+                attr_val=CHANNEL_RASTER[i],
+            )
+        )
+
+        client.SetAttributeI32(
+            nirfmxnr_types.SetAttributeI32Request(
+                instrument=instr,
+                selector_string=subblock_string,
+                attribute_id=nirfmxnr_types.NIRFMXNR_ATTRIBUTE_NUMBER_OF_COMPONENT_CARRIERS,
+                attr_val_raw=NUMBER_OF_COMPONENT_CARRIERS,
+            )
+        )
+
+        for j in range(NUMBER_OF_COMPONENT_CARRIERS):
+            build_carrier_string_response = client.BuildCarrierString(
+                nirfmxnr_types.BuildCarrierStringRequest(
+                    selector_string=subblock_string,
+                    carrier_number=j,
+                )
+            )
+            check_for_warning(build_carrier_string_response, instr)
+            carrier_string = build_carrier_string_response.selector_string_out
+
+            client.SetAttributeF64(
+                nirfmxnr_types.SetAttributeF64Request(
+                    instrument=instr,
+                    selector_string=carrier_string,
+                    attribute_id=nirfmxnr_types.NIRFMXNR_ATTRIBUTE_COMPONENT_CARRIER_BANDWIDTH,
+                    attr_val=COMPONENT_CARRIER_BANDWIDTH[i][j],
+                )
+            )
+
+            client.SetAttributeF64(
+                nirfmxnr_types.SetAttributeF64Request(
+                    instrument=instr,
+                    selector_string=carrier_string,
+                    attribute_id=nirfmxnr_types.NIRFMXNR_ATTRIBUTE_COMPONENT_CARRIER_FREQUENCY,
+                    attr_val=COMPONENT_CARRIER_FREQUENCY[i][j],
+                )
+            )
+
+            client.SetAttributeI32(
+                nirfmxnr_types.SetAttributeI32Request(
+                    instrument=instr,
+                    selector_string=carrier_string,
+                    attribute_id=nirfmxnr_types.NIRFMXNR_ATTRIBUTE_CELL_ID,
+                    attr_val=CELL_ID[i][j],
```

Comment:
I think you'll need to change this to `attr_val_raw=` to appease mypy. It should work across the wire the same I believe.